### PR TITLE
Task #221331: [WEB] PC Users > create prerak list and profile view

### PR DESCRIPTION
**Pull request by snehalsabade**
https://tracker.tekdi.net/issues/221331

### I have ensured that following `Pull Request Checklist` is taken care of before sending this PR
* [] Code is formatted as per format decided
* [] Updated acceptance criteria in tracker
* [] Updated test cases in test-cases-tracker

## Summary by CodeRabbit

- **New Features**
  - Added new translation keys for both English and Hindi locales to support additional UI elements related to certification details, facilities, and user profiles.

- **Refactor**
  - Renamed import `PcUsers` to `PcUsersRoutes` and updated role checks to `program_coordinator` for better clarity and role management.
  - Introduced `PrerakList` and `PrerakProfileView` components to handle user list and profile views efficiently.
  - Updated route paths in the `Footer` component to reflect new navigation structure.

- **Bug Fixes**
  - Adjusted logic for setting selected menu items based on new route paths in the `Footer` component.

### CODE REVIEW

File: lib/common-lib/src/components/PCusers_layout/Footer.js [hunk 1]
```diff
@@ -18,12 +18,12 @@ export default function Footer({ menues, routeDynamics, setRef, ...props }) {
...
     {
       title: 'PRERAKS',
-      route: '/beneficiary/list',
+      route: '/prerak/PrerakList',
       icon: 'GraduationCap'
     },
     {
       title: 'LEARNERS',
-      route: '/beneficiary/list',
+      route: '/prerak/PrerakProfileView',
```
Comment (swapnilphalke03) [inline lib/common-lib/src/components/PCusers_layout/Footer.js:26]: replace "/prerak/PrerakProfileView" with /beneficiary/list

File: apps/front-end/public/locales/hi/translation.json [hunk 2]
```diff
@@ -1650,6 +1654,7 @@
   "VIEW_ALL": "सभी को देखें",
   "VIEW_ATTENDANCE": "उपस्थिति देखें",
   "VIEW_CERTIFICATE": "प्रमाणपत्र देखें",
+  "VIEW_&_DOWNLOAD": "डाउनलोड देखें",
   "VIEW_JOURNEY": "शिक्षार्थी की यात्रा देखेंs",
```
[the remark this answers is not in the document]
Comment (snehalsabade) [reply apps/front-end/public/locales/hi/translation.json:1658]: resolved

File: apps/front-end/src/routes/PcUsersRoutes.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  {
+    path: "prerak/prerakList",
+    component: PrerakList,
+  },
+  {
+    path: "prerak/prerakProfileView/:id",
+    component: PrerakProfileView,
+  },
+  { path: "/", component: PcDashboard },
+  { path: "*", component: PcDashboard },
```
[the remark this answers is not in the document]
Comment (snehalsabade) [reply apps/front-end/src/routes/PcUsersRoutes.js:27]: not required

File: apps/front-end/src/routes/PcUsersRoutes.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+// PC users Routes
+
+export default [
+  {
+    path: "/pcUserDashboard",
+    component: PcDashboard,
+  },
+  {
+    path: "prerak/prerakList",
```
Comment (swapnilphalke03) [inline apps/front-end/src/routes/PcUsersRoutes.js:19]: Replace path with path: "/prerak/prerakList",

File: apps/front-end/src/routes/PcUsersRoutes.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    component: PrerakList,
+  },
+  {
+    path: "prerak/prerakProfileView/:id",
```
Comment (swapnilphalke03) [inline apps/front-end/src/routes/PcUsersRoutes.js:23]: Replace path with path: "/prerak/prerakProfileView/:id",

**Review by swapnilphalke03 — CHANGES_REQUESTED** [review]

File: lib/common-lib/src/components/PCusers_layout/Footer.js (hunk 1 above)
Comment (snehalsabade) [reply lib/common-lib/src/components/PCusers_layout/Footer.js:26]: resolved

File: apps/front-end/src/routes/PcUsersRoutes.js (hunk 5 above)
Comment (snehalsabade) [reply apps/front-end/src/routes/PcUsersRoutes.js:23]: resolved

File: apps/front-end/src/routes/PcUsersRoutes.js (hunk 4 above)
Comment (snehalsabade) [reply apps/front-end/src/routes/PcUsersRoutes.js:19]: resolved

File: apps/front-end/src/pages/PCusers/Prerak/PrerakList.js [hunk 6]
```diff
@@ -0,0 +1,251 @@
...
+        >
+          <List data={data} />
+        </InfiniteScroll>
+      ) : (
+        // Loading component here if needed
+        <></>
+      )}
+    </Layout>
+  );
+}
```
[the remark this answers is not in the document]
Comment (snehalsabade) [reply apps/front-end/src/pages/PCusers/Prerak/PrerakList.js:246]: not required

**Review by swapnilphalke03 — APPROVED** [review]

**Pull request merged**